app/helpers/mailer_helper.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module MailerHelper
+
+  include ApplicationHelper
+
+  def determine_logo_url
+    host      = confirmation_url(Account.first) # TODO: need way to determine a host correctly, just tip to me, but it works!
+    websites  = Website.all
+    logo_url  = nil
+    websites.each do |website|
+      logo_url = website.logo if host.include?(website.domain)
+    end
+    domain_logo_tag(logo_url)
```

Comment:
domain_logo_tag loads white logo. So we either should load black logo or change background color to darker.